### PR TITLE
[plugin.video.yelo@leia] 1.0.1

### PR DESCRIPTION
**Pull request by shycoderX**
### Add-on details:

- **General**
  - Add-on name: Yelo
  - Add-on ID: plugin.video.yelo
  - Version number: 1.0.1
  - Kodi/repository version: leia

- **Code location**
  - URL: https://github.com/add-ons/plugin.video.yelo
  
This plugin makes it possible to watch live streams that are available on the Telenet Yelo application for your region.

### Description of changes:

v1.0.1 (2020-11-02)
- Add Kodi 19 checks @shycoderX)
- Load EPG in the background (@shycoderX)
- Add device registration for Play subscription (@fripsy)
- Use sock.sendall instead of sock.send (@michaelarnauts)

### v1.0.0 (2020-05-22)
- Add Dutch translations (@dagwieers)
- Add support for IPTV Manager (@shycoderX, @dagwieers)
        

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.yelo/.github/workflows/addon-check.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# This workflow performs the required add-on checks for a submission
```
Comment (enen92) [inline plugin.video.yelo/.github/workflows/addon-check.yml]: Please drop from the submission. If using ci/cd add it to gitattributes as export-ignore

File: plugin.video.yelo/.github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# This workflow performs the various sanity tests, integration tests and
```
Comment (enen92) [inline plugin.video.yelo/.github/workflows/ci.yml]: Same

File: plugin.video.yelo/.github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
+# This workflow performs an automated release to GitHub and to the official
```
Comment (enen92) [inline plugin.video.yelo/.github/workflows/release.yml]: Same

File: plugin.video.yelo/addon.xml [hunk 4]
```diff
@@ -26,10 +26,21 @@
         <platform>all</platform>
         <license>GNU General Public License, v3</license>
         <email>shycoderX@protonmail.com</email>
-        <source>https://github.com/shycoderX/repo-plugins/tree/leia/plugin.video.yelo</source>
+        <source>https://github.com/add-ons/plugin.video.yelo</source>
```
Comment (enen92) [inline plugin.video.yelo/addon.xml:29]: Can you change the license to a spdx compatible identifier? Should be GPL-3.0-only

**Review by enen92 — CHANGES_REQUESTED** [review]

File: plugin.video.yelo/resources/lib/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Readme
```
Comment (michaelarnauts) [inline plugin.video.yelo/resources/lib/README.md]: You should also remove this file.

**Comment by shycoderX** [comment]
Ok now?

**Review by enen92 — APPROVED** [review]

**Pull request merged**